File: JavaScript/Q18.js
let month = 2;
switch (month) {
  case 1:
  case 3:
  case 5:
  case 7:
  case 8:
  case 10:
  case 12:
    console.log("Number of days: 31");
    break;
  case 4:
  case 6:
  case 9:
  case 11:
    console.log("Number of days: 30");
    break;
  case 2:
    console.log("Number of days: 28 or 29");
    break;
  default:
    console.log("Invalid month number.");
}

File: Python/Q18.py
month = 2
days_in_month = {
    1: 31, 2: "28 or 29", 3: 31, 4: 30, 5: 31, 6: 30,
    7: 31, 8: 31, 9: 30, 10: 31, 11: 30, 12: 31
}
print(f"The number of days in this month is {days_in_month.get(month, 'Invalid month number.')}")